factories/fake.json

```diff
@@ -79,6 +79,15 @@
                                     "value": "da87d8aa-98f1-4fc3-a260-f92c116fc926"
                                 },
                                 "network_infos": [
+                                    {
+                                        "ip_addresses": [
+                                            {
+                                                "protocol": "IPv4",
+                                                "ip_address": "12.0.1.3"
+                                            }
+                                        ],
+                                        "name": "dcos6"
```

Comment:
Technically not a problem, but lulz. ;)

Reply:
The network name? That's actually the real default ipv6 network name in dcos :-) https://github.com/dcos/dcos/blob/270cb17ec173dc52e69892c1c3ba796414f46936/gen/calc.py#L1140

Reply:
Right, but that's not an ipv6 address ;)

Reply:
Whoops, missed that!